src/py-opentimelineio/opentimelineio/core/__init__.py

```diff
@@ -134,6 +155,45 @@ def wrapped_update(data):
     return decorator_func
 
 
+def downgrade_function_from(cls, version_to_downgrade_from):
+    """
+    Example:
+
+    .. code-block:: python
+
+        @downgrade_function_from(MyClass, 5)
+        def downgrade_from_five_to_four(data):
+            return {"old_attr": data["new_attr"]}
+
+    This will get called to downgrade a schema of MyClass from version 5 to
+    version 4. MyClass must be a class deriving from
+    :class:`~SerializableObject`.
+
+    The downgrade function should take a single argument - the dictionary to
+    downgrade, and return a dictionary with the fields downgraded.
+
+    :param type cls: class to downgrade
```

Comment:
Should the type be `typing.Type[SerializableObject]`? (see https://docs.python.org/3/library/typing.html#typing.Type for the documentation on `typing.Type`)